jumble/deno.json

```diff
@@ -6,6 +6,10 @@
     "test": "echo 'No tests to run.'",
     "integration": "deno test -A ./integration/*.test.ts"
   },
+  "compilerOptions": {
+    "jsx": "react-jsxdev",
+    "types": ["./src/global.d.ts", "./src/vite-env.d.ts"]
+  },
```

Comment:
@jsantell we should probably move these out, but not sure how / whether it's fine to make these apply to every package.

Reply:
We can only use a single `compilerOptions` for the whole workspace unfortunately -- I think this would have to move into the workspace `deno.jsonc`

Reply:
yikes! This might mean we can't use both react JSX and our own JSX? (at compile time) - if I'm reading you correctly @jsantell